app/grandchallenge/reader_studies/urls.py

```diff
@@ -96,6 +103,11 @@
         AddDisplaySetsToReaderStudy.as_view(),
         name="add-displaysets",
     ),
+    path(
+        "<slug>/display-sets/<uuid:pk>/<interface_pk>/images/update/",
```

Comment:
Is `slug` necessary here? You can get the reader study through the display set. I would have intuitively put this with the other display-set urls above.